### PR TITLE
Stories/new designs 179904096

### PR DESCRIPTION
**Pull request by jgoley**
### [PT Story: Implement style guide version of new designs (#179904096)](https://www.pivotaltracker.com/story/show/179904096)

**Tasks:**

- [x] Upcoming appointments calendar (harvest-abt-bhchp, harvest-care-coordinator-dashboard)
- [x] Upcoming appointments calendar tooltip content formatting
- [x] ~~Recent stays timeline ((harvest-abt-bhchp, harvest-care-coordinator-dashboard))~~
- [x] ~~Recent stays timeline tooltip~~
- [x] TPC colored table (harvest-tpc-coc-performance-reports)
- [x] TPC big number with arrow section (harvest-tpc-coc-performance-reports)
- [x] TPC modal with colored header (and structured header) (harvest-tpc-coc-performance-reports)
- [x] MA public dashboard overlay tooltips (harvest-ma-hmis-data-warehouse-2020-2021 public and ad hoc reporting)
- [x] Typography updates (patient card internal card titles and values)

### CODE REVIEW

File: app/assets/stylesheets/application/_settings/_variables.scss [hunk 1]
```diff
@@ -74,3 +74,5 @@ $input-disabled-bg: $gray-025 !default;
 
 // Cards
 $card-padding: 16px !default;
+
+$indicator-colors: #decbe4, #cbd5e8, #e6f5c9;
```
Comment (eanders) [inline app/assets/stylesheets/application/_settings/_variables.scss]: Should these get "themed" in some way, or are they safe for every installation?

Comment (jgoley) [reply app/assets/stylesheets/application/_settings/_variables.scss]: I think those probably could be generic? Unless they clash with other colors in a report? I suppose theming would open up possibilities, we could default to those unless they are explicitly declared in the theme.
The naming is probably off too — I suppose it's more of an "indicator group"?

Comment (eanders) [reply app/assets/stylesheets/application/_settings/_variables.scss]: As long as we can easily override in https://github.com/greenriver/hmis-warehouse/tree/production/app/assets/stylesheets/application/_custom then  I think we're good as is.  Looks like that should be possible.

Comment (jgoley) [reply app/assets/stylesheets/application/_settings/_variables.scss]: 👍 I added the `!default` flag so they should be open to being overridden now.

File: app/models/cohort_columns/delete.rb [hunk 2]
```diff
@@ -36,7 +36,7 @@ def display_for(user)
 
     def display_read_only(_user)
       content_tag(:a, href: pre_destroy_cohort_cohort_client_path(cohort, cohort_client), class: 'btn btn-danger btn-sm btn-icon-only', data: { loads_in_pjax_modal: true }) do
-        content_tag(:i, '', class: 'icon-cross')
+        content_tag(:i, '', class: 'icon icon-cross')
```
Comment (eanders) [inline app/models/cohort_columns/delete.rb]: @jgoley was this change intentional?  I think we have a bunch of `icon-cross` throughout the site with no extra `icon`.  (I feel like we removed that requirement at some point.

File: app/views/style_guides/reports.haml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+      font-weight: bold;
+    }
+    .c-indicator__value-unit {
+      font-size: 16px;
+    }
+  }
+  .c-indicator__label {
+    font-size: 15px;
+    line-height: 16px;
+  }
```
Comment (eanders) [inline app/views/style_guides/reports.haml]: @jgoley shouldn't all of this be in the main CSS?  (I'm confused why it's inline here)

File: app/models/cohort_columns/delete.rb (hunk 2 above)
Comment (jgoley) [reply app/models/cohort_columns/delete.rb]: I think I was trying to give a general icon class but I think we have css that matches "icon" at the start of the class name string. Will remove.

File: app/views/style_guides/reports.haml (hunk 3 above)
Comment (jgoley) [reply app/views/style_guides/reports.haml]: My bad — I inlined it for quicker iterations. I'll drop it in the styles as a component partial.

**Comment by jgoley** [comment]
@eanders I took care of those issues you pointed out. Thanks for the review and feedback.
I merged in production and looks like several commits strangely came along. Any ideas what's happening there?

**Comment by eanders** [comment]
I'm not sure what's up with those commits.  I'll try and sort that.  Any chance you did a force push in your history here?

**Comment by jgoley** [comment]
No force pushes. Only thing that I noticed was I made changes before pulling your commit. But I just pulled with rebase maybe that messed things up?

**Comment by eanders** [comment]
I think the history is funky, but the code changes are correct.  So I'm not concerned.  Thanks!
